File: 搜索插入位置.cpp
#include<stdio.h>
int searchInsert(int* nums, int numsSize, int target) {
    int i=0;
    while(i<numsSize){
        if(nums[i]<target)
            i++;
        else
            return i;
    }
    return i;
}
int main(void){
	int s;
	int nums[5]={1,2,3,4,5};
	s=searchInsert(nums,5,6);
	printf("%d\n",s);
} 

File: 2020年11月/搜索插入位置.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        #第一种方法
        nums.append(target)
        nums = sorted(nums)
        return nums.index(target)

        #第二种方法
        # for i in range(len(nums)):
        #     if nums[i]>=target:
        #         return i
        # return len(nums)